Dockerfile

```diff
@@ -1,56 +1,33 @@
 # syntax=docker/dockerfile:1
+FROM node:20-alpine
 
-# Development stage
-FROM node:20-alpine AS development
 WORKDIR /app
 
-# Install dependencies
 COPY package*.json ./
-COPY prisma ./prisma/
-
-# Install all dependencies (including devDependencies)
-RUN npm install
-
-# Generate Prisma client
-RUN npx prisma generate
-
-# Production builder stage
-FROM node:20-alpine AS builder
-WORKDIR /app
+RUN npm ci
 
-# Install dependencies
-COPY package*.json ./
 COPY prisma ./prisma/
-RUN npm install
+COPY src ./src
+COPY public ./public
```

Comment:
**issue:** We need to remove the `public` folder here, as it does not exist before building.